config/configvalidator.php

```diff
@@ -48,32 +48,18 @@ public function isConfigSafe($key, $parsedConfigItem) {
 	 *
 	 * @param array $parsedConfigItem the sorting configuration to analyse
 	 * @param bool $safe whether the current config has been deemed safe to use so far
-	 *
+	 * @param string key  whether this exist or not 
```

Comment:
typo and could be more explicit. Think about the other developers who will read your code, they need to be able to quickly understand what is going on by reading the documentation.